qemu/target-i386/cpu.h

```diff
@@ -699,7 +699,7 @@ typedef enum {
 
 typedef struct SegmentCache {
     uint32_t selector;
-    target_ulong base;
+    uint64_t base;   /* handle 32 or 64 bit CPUs */
```

Comment:
why do we need to copy `x86_mmr` from `include/unicorn/x86.h` to somewhere else? we can just make sure `unicorn.c` can reference this struct in `x86.h`, can't we?

btw, for naming consistency, please stick "uc_" in front of `x86_mmr` struct.